core/domain/classifier_services.py

```diff
@@ -153,6 +153,75 @@ def handle_non_retrainable_states(exploration, state_names, exp_versions_diff):
         job_exploration_mappings)
 
 
+FLOAT_INDICATOR_KEY = 'float_values'
```

Comment:
Add comment pointing to the instance of this in oppia/oppia-ml and saying that the two must be the same. Might also be better to put it in feconf since it kind of is configuration for the oppia app.

Reply:
Done